src/lib/sql/sql_identifier_resolver.cpp

```diff
@@ -48,19 +53,24 @@ std::shared_ptr<AbstractExpression> SQLIdentifierResolver::resolve_identifier_re
   return matching_expressions[0];
 }
 
-const std::optional<SQLIdentifier> SQLIdentifierResolver::get_expression_identifier(
-    const std::shared_ptr<AbstractExpression>& expression) const {
+const std::vector<SQLIdentifier> SQLIdentifierResolver::get_expression_identifiers(
+    const std::shared_ptr<AbstractExpression> &expression) const {
```

Comment:
The position of the ampersand looks like the linter/formatter haven't been executed.